{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-15 20:10:55"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-15 20:10:55"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-15 20:10:55"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-15 20:35:34"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-15 20:35:34"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-15 20:35:34"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-15 20:37:02"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-15 20:37:02"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-15 20:37:02"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-15 20:37:07"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-15 20:37:07"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-15 20:37:07"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-15 20:37:14"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-15 20:37:14"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-15 20:37:14"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-15 20:38:13"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-15 20:38:13"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-15 20:38:13"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-15 20:39:49"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-15 20:39:49"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-15 20:39:49"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-15 20:39:52"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-15 20:39:52"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-15 20:39:52"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-15 20:47:27"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-15 20:47:27"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-15 20:47:27"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-15 20:49:44"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-15 20:49:44"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-15 20:49:44"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-15 20:49:48"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-15 20:49:48"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-15 20:49:48"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-15 20:49:52"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-15 20:49:52"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-15 20:49:52"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-15 20:52:59"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-15 20:52:59"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-15 20:52:59"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-15 20:53:04"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-15 20:53:04"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-15 20:53:04"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-15 20:53:18"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-15 20:53:18"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-15 20:53:18"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-15 20:53:33"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-15 20:53:33"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-15 20:53:33"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-15 20:53:50"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-15 20:53:50"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-15 20:53:50"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-15 20:54:00"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-15 20:54:00"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-15 20:54:00"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-15 20:54:22"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-15 20:54:22"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-15 20:54:22"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-15 20:54:57"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-15 20:54:57"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-15 20:54:57"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-15 20:55:02"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-15 20:55:02"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-15 20:55:02"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-15 20:55:55"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-15 20:55:55"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-15 20:55:55"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-15 20:56:04"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-15 20:56:04"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-15 20:56:04"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-15 20:56:41"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-15 20:56:42"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-15 20:56:42"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-15 20:56:47"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-15 20:56:47"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-15 20:56:47"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-15 20:57:08"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-15 20:57:08"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-15 20:57:08"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-15 20:57:42"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-15 20:57:43"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-15 20:57:43"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-15 20:58:14"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-15 20:58:14"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-15 20:58:14"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-15 20:58:30"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-15 20:58:30"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-15 20:58:30"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-15 20:59:03"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-15 20:59:03"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-15 20:59:03"}
